packages/mux-uploader/src/mux-uploader-sr-text.ts

```diff
@@ -0,0 +1,59 @@
+import { globalThis, document } from 'shared-polyfills';
+import { getMuxUploaderEl } from './utils/element-utils';
+
+const template = document.createElement('template');
+
+template.innerHTML = `
+<style>
+
+.sr-only {
+  position:absolute;
+  left:-10000px;
+  top:auto;
+  width:1px;
+  height:1px;
+  overflow:hidden;
+}
+</style>
+
+<div class="sr-only" id="sr-only" aria-live="polite"></div>
+`;
+
+class MuxUploaderSrTextElement extends globalThis.HTMLElement {
+  srOnlyText: HTMLElement | null | undefined;
+  #uploaderEl: HTMLElement | null | undefined;
+
+  constructor() {
+    super();
+    const shadowRoot = this.attachShadow({ mode: 'open' });
+    shadowRoot.appendChild(template.content.cloneNode(true));
+
+    this.srOnlyText = this.shadowRoot?.getElementById('sr-only');
+  }
+
+  connectedCallback() {
+    this.#uploaderEl = this.shadowRoot?.host ? getMuxUploaderEl(this.shadowRoot.host) : null;
```

Comment:
question(blocking): Can we just use `getMuxUploaderEl(this)`? if yes, let's simplify.